### PR TITLE
Unit test for AxisGuesser

### PR DESCRIPTION
**Pull request by simleo**
This PR adds a new unit test for `AxisGuesser`.

### CODE REVIEW

File: components/formats-bsd/test/loci/formats/utests/AxisGuesserTest.java [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+      }
+    }
+    assertEquals(ag.getAxisCount(Z_AXIS), countZ);
+    assertEquals(ag.getAxisCountZ(), countZ);
+    assertEquals(ag.getAxisCount(T_AXIS), countT);
+    assertEquals(ag.getAxisCountT(), countT);
+    assertEquals(ag.getAxisCount(C_AXIS), countC);
+    assertEquals(ag.getAxisCountC(), countC);
+    assertEquals(ag.getAxisCount(S_AXIS), countS);
+    assertEquals(ag.getAxisCountS(), countS);
```
Comment (melissalinkert) [inline components/formats-bsd/test/loci/formats/utests/AxisGuesserTest.java]: Worth asserting that ```ag.getAxisCount(UNKNOWN_AXIS)``` is 0?

**Comment by melissalinkert** [comment]
It looks like the test isn't being run during ```ant test```; it probably needs to be added to ```components/formats-bsd/test/loci/formats/utests/testng.xml```.

**Comment by simleo** [comment]
I've just opened #2190 to bring in more recently added tests.

**Comment by simleo** [comment]
The `BUILD=maven` Travis job hangs at the start of the test phase. It's already been restarted, but it failed again, in the same way. It runs fine on my laptop.

**Comment by melissalinkert** [comment]
Travis is certainly green now, and everything looks good locally.  Merging.

**Pull request merged**